core/filter_api.php

```diff
@@ -3186,6 +3205,10 @@ function filter_draw_selection_inputs( $p_filter, $p_for_screen = true, $p_stati
  *      These functions could also be shared by view_filters_page.php
  */
 
+/**
+ * Print the current value of this filter field, as visible string, and as a hidden form input.
+ * @param type $p_filter	Filter array
+ */
 function print_filter_values_reporter_id( $p_filter ) {
```

Comment:
For methods that return no value, add "@return void".  Applies to multiple methods.